dev-lang/ispc/ispc-9999.ebuild

```diff
@@ -35,9 +35,9 @@ DEPEND="
 	"
 
 PATCHES=(
-	"${FILESDIR}/${PN}-1.13.0-cmake-gentoo-release.patch"
-	"${FILESDIR}/${PN}-1.13.0-llvm-10.patch"
-	"${FILESDIR}/${PN}-1.13.0-werror.patch"
+	"${FILESDIR}/${PN}-1.14.0-cmake-gentoo-release.patch"
+	"${FILESDIR}/${PN}-1.14.0-llvm-10.patch"
+	"${FILESDIR}/${PN}-1.14.0-werror.patch"
```

Comment:
It looks like the cmake + werror patches didn't change at all, just bumped the version number. We normally leave these as-is unless they got rebased.